terraform-infra-approvals/idam-web-public.json

```diff
@@ -1,7 +1,8 @@
 {
   "resources": [],
   "module_calls": [
-    {"source":  "git@github.com:hmcts/cnp-module-webapp?ref=SIDM-3089"}
+    {"source":  "git@github.com:hmcts/cnp-module-webapp?ref=SIDM-3089"},
+    {"source":  "git@github.com:hmcts/cnp-module-redis?ref=azurerm_redis_cache-resource"}
```

Comment:
this need doing now? should be testable now and then hopefully we can just merge it

Reply:
We're trying to test in sandbox but having a few issues with builders. Once it's passed, yes but doesn't need merging in a rush

Reply:
tbh I would have just tested in isolation from your machine, pretty straight forward test...